samples/wcfapp/README.md

```diff
@@ -0,0 +1,59 @@
+# WCF Docker Container Samples
```

Comment:
I would like to see us build and push at least one sample to microsoft/dotnet-framework-samples so that someone can just pull and try something.  e.g. https://github.com/Microsoft/dotnet-framework-docker/blob/master/samples/dotnetapp/README.md#try-a-pre-built-net-framework-docker-image

To do this you can add the appropriate entries in https://github.com/Microsoft/dotnet-framework-docker/blob/master/manifest.samples.json.  I can help you with this if you have questions.